src/dsp_tools/commands/ingest_xmlupload/bulk_ingest_client.py

```diff
@@ -151,4 +151,4 @@ def retrieve_mapping_generator(self) -> Iterator[str | bool]:
             else:
                 logger.info("Ingest process completed.")
                 break
-        yield res.text
+        yield res.content.decode("utf-8")
```

Comment:
This is the fix of https://linear.app/dasch/issue/DEV-4156/bulk-ingest-incorrectly-encodes-special-chars.

`res.text` returns `sÃ¼d.png`
`res.content.decode("utf-8")`  returns `süd.png`